python/ray/air/callbacks/keras.py

```diff
@@ -1,4 +0,0 @@
-from ray.tune._structure_refactor import warn_structure_refactor
-from ray.air.integrations.keras import *  # noqa: F401, F403
-
-warn_structure_refactor(__name__, "ray.air.integrations.keras")
```

Comment:
@krfricke did we ever establish a cleanup policy for `warn_structure_refactor`? Could we clean up the rest of these as well (in a separate PR)?